python-main.js

```diff
@@ -356,6 +356,15 @@ function web_editor(config) {
         } else {
             return alert(config.translate.alerts.empty);
         }
+        
+        if (micropythonFs.getStorageRemaining() < 0){
+            alert(config.translate.alerts.out_of_space);
+            if (isModule){
+                micropythonFs.remove(filename);
+                return alert(config.translate.alerts.module_out_of_space);
+            }
+        }
+        
```

Comment:
A nitpick, but we can remove the whitespaces before and after the `if` statement.